static/js/components/auth/AuthEmailForm.js

```diff
@@ -21,7 +21,7 @@ const AuthEmailForm = ({
   processing,
   formError
 }: LoginFormProps) => {
-  const suggestion = validEmail(form.email) ? suggestEmail(form.email) : null
+  const suggestion = !isEmptyText(form.email) ? suggestEmail(form.email) : null
```

Comment:
Do we still need the ternary here? Just wondering b/c it looks like `suggestEmail` returns `null` if it doesn't have a valid suggestion to offer